packages/synapse-react-client/src/components/QueryWrapper/useHasFacetedSelectColumn.ts

```diff
@@ -0,0 +1,19 @@
+import { QueryContextType } from '../QueryContext'
+import { useQuery } from '@tanstack/react-query'
+import { hasFacetedSelectColumn as hasFacetedSelectColumnUtil } from '../../utils/functions/queryUtils'
+
+export default function useHasFacetedSelectColumn(
```

Comment:
New custom hook with logic that was previously in QueryWrapper, extracted to make QueryWrapper as lean as possible.